rakelib/kokoro/kokoro.rb

```diff
@@ -58,6 +59,23 @@ def continuous
     release_please if @should_release && @updated
   end
 
+  def cloudrad
+    run_ci @gem do
+      header "Building Doc YAMLs"
+      FileUtils.remove_dir "doc" if Dir.exists? "doc"
+      run "bundle exec rake cloudrad", 1800
+      RepoMetadata.from_source(".repo-metadata.json").build "."
```

Comment:
Do we need to build this into the "doc" directory? (And run docuploader from the doc directory?) Or is cloud-rad different?